### DISCUSSION TITLE
Need help to get a megaboom gate level netlist

### DISCUSSION THREAD

**Discussion by xtofalex** [The-OpenROAD-Project/megaboom#28] [Q&A]
Hi everyone,
I'm trying to access big open source designs (the biggest, the better) for my own project and megaboom seems a very good fit.
I'm looking for having access to a gate level netlist but have not been able to run synthesis and would appreciate some help.
I'm not a bazel specialist and not using Google Cloud. I did not understand from the README if it is mandatory or not.
Can someone guide me in order to get at least a synthesis result ?
Thanks in advance !
Christophe

**Comment by oharboe** [comment]
Google Cloud is not mandatory, you use it if you want to share artifacts(build results).

**Reply by xtofalex** [reply]
Ok.
Sorry again for the bazel complete newbie question.
I cloned https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts in my home directory.
When launching:
bazelisk build L1MetadataArray_test_generate_abstract
I get:
ERROR: /home/xtof/megaboom/WORKSPACE.bazel:15:17: fetching local_repository rule //external:bazel-orfs: java.io.IOException: The repository's path is "../bazel-orfs" (absolute: "/home/xtof/bazel-orfs") but it does not exist or is not a directory.
WARNING: Target pattern parsing failed.
ERROR: Skipping 'L1MetadataArray_test_generate_abstract': no such package '@@bazel-orfs//': The repository's path is "../bazel-orfs" (absolute: "/home/xtof/bazel-orfs") but it does not exist or is not a directory.
ERROR: no such package '@@bazel-orfs//': The repository's path is "../bazel-orfs" (absolute: "/home/xtof/bazel-orfs") but it does not exist or is not a directory.

**Reply by oharboe** [reply]
You need to clone https://github.com/The-OpenROAD-Project/bazel-orfs alongside megaboom.

@lpawelcz Note that I'm hoping https://github.com/The-OpenROAD-Project/bazel-orfs/pull/16 with the corresponding pull request for megaboom will land in the next few days... This hopefully will start to make things simpler and improve documentation.
